exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetGroupScan.java

```diff
@@ -115,6 +115,8 @@
   private List<RowGroupInfo> rowGroupInfos;
   private Metadata.ParquetTableMetadataBase parquetTableMetadata = null;
   private String cacheFileRoot = null;
+  private int batchSize;
+  private static final int DEFAULT_BATCH_LENGTH = 256 * 1024;
```

Comment:
If it has nothing to do with the new option, why do they have the same value of 256 * 1024? Just  coincidence? 

Also, since a new option is introduced, if user alters this option and set it to a different value than the default, will the option be honored in parquet reader?